.github/scripts/setup_runners_service.sh

```diff
@@ -240,7 +238,7 @@ then
   elif [ "$SETUP_DISTRO" = "docker" ]
   then
     docker restart $DOCKER_NAME
-    docker run -d -p 5601:5601 --name $DOCKER_NAME_KIBANA --network="host" odfe-kibana-http:security
+    docker run -d --name $DOCKER_NAME_KIBANA --network="host" odfe-kibana-http:security
```

Comment:
Does it still work without exposing the Kibana port? If so, how?

Reply:
You should not / need not to use port binding anymore if you use host network.

Reply:
OK, cool.